core/src/avm1/globals/xml.rs

```diff
@@ -222,6 +222,8 @@ impl<'gc> Xml<'gc> {
             }
         }
 
+        self.root().refresh_cached_child_nodes(activation).unwrap(); // :(
```

Comment:
I don't think the AVM1 operations done by `refresh_cached_child_nodes` can throw an error?